experimental/generation/generator/packages/library/test/merge_data/modified/language-understanding/en-us/Bread/sandwichMerge-Bread-BreadValue.en-us.lu

```diff
@@ -60,120 +60,120 @@
 > Assign examples for operations on the 'Bread' property to the intent 'sandwich'.
 # sandwich
 >> entity: Standalone BreadProperty with BreadValue
-- {@BreadProperty={@BreadValue=multi}}
-- {@BreadProperty={@BreadValue=grain}}
-- {@BreadProperty={@BreadValue=wheat}}
-- {@BreadProperty={@BreadValue=multi grain}}
-- {@BreadProperty={@BreadValue=grain wheat}}
-- {@BreadProperty={@BreadValue=multi grain wheat}}
->- {@BreadProperty={@BreadValue=rye}}
-- {@BreadProperty={@BreadValue=white}}
-- {@BreadProperty={@BreadValue=whole}}
-- {@BreadProperty={@BreadValue=whole wheat}}
-- {@BreadProperty={@BreadValue=multi} bread}
-- {@BreadProperty={@BreadValue=grain} bread}
-- {@BreadProperty={@BreadValue=wheat} bread}
-- {@BreadProperty={@BreadValue=multi grain} bread}
-- {@BreadProperty={@BreadValue=grain wheat} bread}
-- {@BreadProperty={@BreadValue=multi grain wheat} bread}
->- {@BreadProperty={@BreadValue=rye} bread}
-- {@BreadProperty={@BreadValue=white} bread}
-- {@BreadProperty={@BreadValue=whole} bread}
-- {@BreadProperty={@BreadValue=whole wheat} bread}
+- {@BreadProperty={BreadValue=multi}}
```

Comment:
>{@ [](http://example.com/codeflow?start=2&length=2)

This should not be there.  This is the file where all labels should not include an @.